bazeldnf/extensions.bzl

```diff
@@ -106,6 +106,8 @@ def _alias_repository_impl(repository_ctx):
 
     if repository_ctx.attr.lock_file.package:
         lock_file_path = repository_ctx.attr.lock_file.package + "/" + lock_file_path
+
+    if repository_ctx.attr.repofile and repository_ctx.attr.repofile.package:
```

Comment:
shame in repository context we don't get ctx.file.repofile like we do in rules, as then we wouldn't have to mess with this, good finding